ur5e_moveit_config/urdf/panther_ur5e.urdf.xacro

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0'?>
+
+<robot name="panther_ur5e" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  
+  <xacro:include filename="$(find ur5e_moveit_config)/urdf/panther_base_macro.urdf.xacro" ns="husarion" />
+  <xacro:include filename="$(find ur5e_moveit_config)/urdf/ur_macro.xacro" ns="universal_robot"/>
+  <xacro:include filename="$(find ur5e_moveit_config)/urdf/external_antena_macro.urdf.xacro" ns="antena" />
+  <xacro:include filename="$(find ur5e_moveit_config)/urdf/power_supply_macro.xacro" ns="power_supply"/>
+  <xacro:include filename="$(find ros_components_description)/urdf/velodyne_puck.gazebo.xacro" ns="velodyne" />
+  <xacro:include filename="$(find realsense2_description)/urdf/_d435i.urdf.xacro" ns="rs"/>
+
+  <!-- !TODO: 
+          when updating the package to the latest version of panther_description, 
+          add rails that reflect the actual components -->
+
+  <!-- <xacro:include filename="$(find ur5e_moveit_config)/urdf/upper_rails.urdf.xacro" ns="rails" /> -->
+  <!-- <xacro:rails.upper_rails parent_link="base_link" xyz="0.0 0.0 0.334" rpy="0.0 ${-pi/2} 0.0"  /> -->
```

Comment:
I guess we can use upper rails as default in this demo. To ensure moveit doesn't see colisions between this element and robot self-collision checking in moveit manager can be used.